README.md

```diff
@@ -169,6 +169,11 @@ Have a look at the [contributing guidelines](CONTRIBUTING.md), and go ahead!
   The examples are submitted by the user base, and can be voted up or down;
   the best entries are what people see first when they look up a command.
 
+- [kb](https://github.com/gnebbia/kb)
+  it is a minimalist command line knowledge base manager.
+  kb can be used to organize your notes and cheatsheets in a minimalist
```

Comment:
I would do "It" more in the style of the others.
```suggestion
  It can be used to organize your notes and cheatsheets in a minimalist
```

Reply:
I think your suggestion above is the better one :-)

Reply:
You mean https://github.com/tldr-pages/tldr/pull/5042#discussion_r547233172 and **not** https://github.com/tldr-pages/tldr/pull/5042#discussion_r547233973?

Reply:
Yep, that's correct. The other suggestion is more consistent with rest of items in the list :-)